Resources/Prototypes/SS220/Reagents/Consumable/Drink/drinks.yml

```
@@ -20,4 +20,21 @@
   color: "#f04737"
   metamorphicSprite:
     sprite: SS220/Objects/Consumable/Drinks/glass_berry.rsi
-    state: icon   
\ No newline at end of file
+    state: icon   
+
+#Bankate
+- type: reagent
+  id: BankateDrink
+  name: reagent-name-bankate
+  parent: BaseDrink
+  desc: reagent-desc-bankate
+  physicalDesc: reagent-physical-desc-strong-smelling
+  flavors: horrible
+  color: "#dfe15f"
+  recognizable: true
+  metamorphicSprite:
+    sprite: SS220/Objects/Consumable/Drinks/glass_bankate.rsi
+    state: icon_empty
+  metamorphicMaxFillLevels: 4
+  metamorphicFillBaseName: fill-
+  metamorphicChangeColor: false
```

```suggestion
  metamorphicChangeColor: false
  reactiveEffects:
    Acidic:
      methods: [ Touch ]
      effects:
      - !type:HealthChange
        scaleByQuantity: true
        ignoreResistances: false
        damage:
          types:
            Caustic: 0.3
      - !type:PopupMessage
        messages: [ "жидкость жжётся!" ]
        visualType: MediumCaution
        probability: 0.3
        type: Local
```
банкате просто как напиток это ведь скучно, зато могли бы использовать в качестве оружия, хоть и слабого